v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -245,6 +245,11 @@ var functions = map[string]govaluate.ExpressionFunction{
 		data := deserialization.GenerateJavaGadget(gadget, cmd, encoding)
 		return data, nil
 	},
+	// for debug purposes
+	"print_stdout": func(args ...interface{}) (interface{}, error) {
+		fmt.Println(args)
```

Comment:
We should add a message before printing the debug values, stating they are from print_debug function. Something like this @yabeow 

```go
	gologger.Info().Msgf("print_stdout debug value: ")
        fmt.Println(args)
```

Reply:
Nice suggestion! I've added this, please check again 👍